server/channels/app/webhook_test.go

```diff
@@ -732,6 +732,110 @@ func TestTriggerOutGoingWebhookWithUsernameAndIconURL(t *testing.T) {
 
 }
 
+func TestTriggerOutGoingWebhookWithMultipleURLs(t *testing.T) {
+	getPayload := func(hook *model.OutgoingWebhook, th *TestHelper, channel *model.Channel) *model.OutgoingWebhookPayload {
+		return &model.OutgoingWebhookPayload{
+			Token:       hook.Token,
+			TeamId:      hook.TeamId,
+			TeamDomain:  th.BasicTeam.Name,
+			ChannelId:   channel.Id,
+			ChannelName: channel.Name,
+			Timestamp:   th.BasicPost.CreateAt,
+			UserId:      th.BasicPost.UserId,
+			UserName:    th.BasicUser.Username,
+			PostId:      th.BasicPost.Id,
+			Text:        th.BasicPost.Message,
+			TriggerWord: "Abracadabra",
+			FileIds:     strings.Join(th.BasicPost.FileIds, ","),
+		}
+	}
+
+	createOutgoingWebhook := func(channel *model.Channel, testCallBackURLs []string, th *TestHelper) (*model.OutgoingWebhook, *model.AppError) {
+		outgoingWebhook := model.OutgoingWebhook{
+			ChannelId:    channel.Id,
+			TeamId:       channel.TeamId,
+			CallbackURLs: testCallBackURLs,
+			Username:     "some-user-name",
+			IconURL:      "http://some-website.com/assets/some-icon.png",
+			DisplayName:  "some-display-name",
+			Description:  "some-description",
+			CreatorId:    th.BasicUser.Id,
+			TriggerWords: []string{"Abracadabra"},
+			ContentType:  "application/json",
+		}
+
+		return th.App.CreateOutgoingWebhook(&outgoingWebhook)
+	}
+
+	type TestCaseOutgoing struct {
+		CallBackURLs []string
+	}
+
+	chanTs1 := make(chan string, 1)
+	ts1 := httptest.NewServer(http.HandlerFunc(func(w http.ResponseWriter, r *http.Request) {
+		chanTs1 <- "webhook received!"
+	}))
+	defer ts1.Close()
+
+	chanTs2 := make(chan string, 1)
+	ts2 := httptest.NewServer(http.HandlerFunc(func(w http.ResponseWriter, r *http.Request) {
+		chanTs2 <- "webhook received!"
+	}))
+	defer ts2.Close()
+
+	getTestCases := func() map[string]TestCaseOutgoing {
+
+		testCasesOutgoing := map[string]TestCaseOutgoing{
+
+			"One WebhookURL": {
+				CallBackURLs: []string{ts1.URL},
+			},
+			"Two WebhookURLs": {
+				CallBackURLs: []string{ts1.URL, ts2.URL},
+			},
+		}
+		return testCasesOutgoing
+	}
+
+	th := Setup(t).InitBasic()
+	defer th.TearDown()
+
+	th.App.UpdateConfig(func(cfg *model.Config) {
+		*cfg.ServiceSettings.AllowedUntrustedInternalConnections = "localhost,127.0.0.1"
+	})
+
+	for name, testCase := range getTestCases() {
+		t.Run(name, func(t *testing.T) {
```

Comment:
We can inline the test cases like this, to keep it more contextual:

```suggestion
	for name, testCase := range map[string]struct{
		CallBackURLs []string
	} {
		"One WebhookURL": {
			CallBackURLs: []string{ts1.URL},
		},
		"Two WebhookURLs": {
			CallBackURLs: []string{ts1.URL, ts2.URL},
		},
	} {
		t.Run(name, func(t *testing.T) {
```